### PR TITLE
Replace ¢ -> $ and Balance -> Balance Used

### LINKED ISSUE #13809: Replace ¢ with $ and clarify Balance in UBP billing settigs UI
- Users will understand $0.10 more easily than 10¢ when they read descriptions of plans in the UI.
- The indicator shows used credits but the word `BALANCE` suggests that it shows remaining credits. How about `BALANCE USED` instead?

### PR DESCRIPTION
**Pull request by jldec**
## Description
Refine wording in Billing settings page based on user feedback.

## Related Issue(s)
Fixes #13809

## How to test
View UBP billing settings for users and teams.

## Release Notes
```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-payment
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by jldec** [comment]
Screenshot 2022-10-14 at 08 51 18
Screenshot 2022-10-14 at 08 52 50
Screenshot 2022-10-14 at 08 53 50
Screenshot 2022-10-14 at 08 54 48

File: components/dashboard/src/components/UsageBasedBillingConfig.tsx [hunk 1]
```diff
@@ -279,7 +279,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                             <Check className="m-0.5 w-5 h-5" />
                             <div className="flex flex-col">
                                 <span>
-                                    36¢ for 10 credits or 1 hour of Standard workspace usage.{" "}
+                                    $0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```
Comment (jankeromnes) [inline components/dashboard/src/components/UsageBasedBillingConfig.tsx:282]: Please make the currency dynamic (based on user region -- later we could improve it to actual subscription currency):

```suggestion
                                    {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```

**Comment by geropl** [comment]
Taking a look now...

File: components/dashboard/src/components/UsageBasedBillingConfig.tsx [hunk 2]
```diff
@@ -386,7 +386,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                                         <Check className="m-0.5 w-5 h-5 text-orange-500" />
                                         <div className="flex flex-col">
                                             <span>
-                                                36¢ for 10 credits or 1 hour of Standard workspace usage.{" "}
+                                                $0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```
Comment (jankeromnes) [inline components/dashboard/src/components/UsageBasedBillingConfig.tsx:389]:
```suggestion
                                                {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard workspace usage.{" "}
```

**Review by jankeromnes — APPROVED** [review]
Code changes look good to me, thanks!

One minor suggestion: Please use a dynamic currency symbol (auto-detected from user region) as opposed to hard-coding `$`. Otherwise, good to go. 🚀 🛹

/hold

**Review by geropl — APPROVED** [review]
LGTM and works.

/hold for addressing @jankeromnes comment [here](https://github.com/gitpod-io/gitpod/pull/13857/files#r995412048).

**Comment by jldec** [comment]
Thanks for the suggestion @jankeromnes - I should have thought of that 👍 :)

**Comment by jankeromnes** [comment]
All nits addressed and still works! Let's ship this 🚀 🛹

/unhold

File: components/dashboard/src/components/UsageBasedBillingConfig.tsx [hunk 3]
```diff
@@ -330,7 +330,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
                                 <Check className="m-0.5 w-5 h-5" />
                                 <div className="flex flex-col">
                                     <span className="font-bold">Pay-as-you-go after 1,000 credits</span>
-                                    <span>36¢ for 10 credits or 1 hour of Standard workspace usage.</span>
+                                    <span>{currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard workspace usage.</span>
```
Comment (easyCZ) [inline components/dashboard/src/components/UsageBasedBillingConfig.tsx:333]:
```
10 credits at {currency === "EUR" ? "€" : "$"}0.36 or equivalent 1 hour of Standard workspace usage.
```

**Pull request merged**